api/app/GraphQL/Queries/Log/LogsQuery.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace App\GraphQL\Queries\Log;
+
+use GraphQL\Type\Definition\Type;
+use Illuminate\Support\Facades\Log;
+use Rebing\GraphQL\Support\Facades\GraphQL;
+use Rebing\GraphQL\Support\Query;
+
+class LogsQuery extends Query
```

Comment:
Could be worth calling this 'JobStateLog' to be more explicit about what's being tracked here.